first-task/src/components/Scroll/ScrollToTop.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useCallback, useEffect, useState } from "react";
+import {useWindowScroll} from "react-use"
+
+const ScrollToTop = () => {
+    const {y: pageYOffset} = useWindowScroll()
+    const [visible, setVisible] = useState(false);
+
+    useEffect(() => {
+        if(pageYOffset > 400){
+            setVisible(true)
+        }else{
+            setVisible(false)
+        }
```

Comment:
you can simplify this construction and write like this  `setVisible(pageYOffset > 400)`